pkg/reconciler/knativeserving/common/images.go

```diff
@@ -109,6 +113,10 @@ func updateCachingImage(instance *servingv1alpha1.KnativeServing, u *unstructure
 	if err != nil {
 		return err
 	}
+	// Cleanup zero-value default to prevent superfluous updates
+	u.SetCreationTimestamp(metav1.Time{})
+	delete(u.Object, "status")
```

Comment:
I was curious about this, too, because the core types don't exhibit this behavior. Note that converting to `appv1.Deployment` did not set a default status. Because `Image` is a knative type, I expect there may be some critical difference between the core and knative generated functions. Could it be a bug with `Image` that it's setting that status field at all?